<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #586380;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>18</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 18pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Nhập lại từ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tu_vung">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>18</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 87 18pt &quot;Segoe UI Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Carbohydrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okey">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#okey{
color: rgb(255, 255, 255);
border-radius: 6px;
background-color: #7583ff;
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#okey:hover {
  background-color: blue;
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Đáp án</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continues">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#continues{
border-radius: 6px;
background-color: rgb(67, 67, 100);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QPushButton#continues:hover {
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
  background-color: rgb(57, 57, 85)
}</string>
    </property>
    <property name="text">
     <string>Bạn đã thuộc?</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="title">
     <string>Dùng</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="title">
     <string>Text</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>15</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Danh Từ --&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>15</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Danh Từ --&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="voice">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QToolButton#voice{
border-radius: 8px;
background-color: rgb(0, 49, 149);
}

QToolButton#voice:hover {
  background-color: blue;
}
QToolButton#voice:pressed {
  background-color: red;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="voice_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QToolButton#voice_2{
border-radius: 8px;
background-color: rgb(208, 69, 0);
}

QToolButton#voice_2:hover {
  background-color: rgb(189, 0, 0);
}
QToolButton#voice_2:pressed {
  background-color: red;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="text">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#text{
border-radius: 8px;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Input...</string>
    </property>
   </widget>
   <widget class="QLabel" name="phien_am">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Phiên âm</string>
    </property>
   </widget>
   <widget class="QLabel" name="loai_tu">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>noun</string>
    </property>
   </widget>
   <widget class="QLabel" name="nghia">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nghiax</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
